### DISCUSSION TITLE
Custom components with content kit works in editor but not in preview.

### DISCUSSION THREAD

**Discussion by gewenyu99** [GitbookIO/community#829] [0. General bugs and questions]
See below, in editor, the component (selector) show up properly:

<img width="782" alt="Screenshot 2024-12-16 at 11 08 07 PM" src="https://github.com/user-attachments/assets/62106f57-db37-4661-9450-fc42ffeb0c71" />

How ever, see that in a preview they show up like this:

<img width="850" alt="Screenshot 2024-12-16 at 11 08 37 PM" src="https://github.com/user-attachments/assets/8a4ae11c-05d3-43c4-afb4-ed6ca6699dda" />

**Comment by addisonschultz** [comment]
Hi @gewenyu99 - thanks for highlighting this. It’s something we’re aware of and have an open ticket for. We’ll try to get some of these blocks implemented soon!

**Reply by gewenyu99** [reply]
@addisonschultz thanks, is this an issue in preview or does it also not work in production?

**Reply by addisonschultz** [reply]
Right now it’s broken in the rendering part of GitBook, for previews and production 😕

**Reply by gewenyu99** [reply]
@addisonschultz Hi friend, are there any updates here? Really looking forward to use this.

**Reply by gewenyu99** [reply]
@addisonschultz Helllooo friend~ Are there any updates?

**Reply by addisonschultz** [reply]
Hello @gewenyu99 - unfortunately not at this time. This has been in our backlog for a bit as we’re working on other high-priority fixes and tasks!
